tjdashboard/code/site/libraries/dashboard.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_truck
```

Comment:
truck ?? Ensure that the file header lines are correct. 

package should be Com_Tjdashboard
you can skip subpackage

Reply:
Done